src/client.ts

```diff
@@ -1,24 +1,52 @@
 import { Client, Intents } from "discord.js";
+import { config } from "dotenv";
 import { Db } from "mongodb";
 
 import cmd_register from "./commands/register";
 import {
   cmd_tenmans,
   handleButton as tenmansHandleButton,
 } from "./commands/tenmans";
+import { cmdConfig } from "./commands/config";
+import botConfig from "./config/botConfig";
+
+async function initBotConfig (client: Client, db: Db) {
+  const botConfigCollection = db.collection("config");
+  let botConfigDoc = await botConfigCollection.findOne({
+    "configName": "bot"
```

Comment:
Same here: I think prettier prefers no quotes on keys unless the name can't be typed out in JS